psalm-baseline.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <files psalm-version="5.11.0@c9b192ab8400fdaf04b2b13d110575adc879aa90">
+  <file src="src/Differ/DiffColorizer.php">
+    <InvalidArrayOffset>
+      <code>$lines[$prevIndex]</code>
```

Comment:
I don't know why PSalm doesn't get the [`Assert::greaterThan($index, 0);`](https://github.com/infection/infection/pull/1974/files#diff-959b4198e0999129d1b54c6c33f931e33184201166012bb135dce6515ba78196R65)